test/test_excerpt.rb

```diff
@@ -78,19 +78,21 @@ def do_render(document)
     context "#to_liquid" do
       should "contain the proper page data to mimick the post liquid" do
         assert_equal "Post Excerpt with Layout", @excerpt.to_liquid["title"]
-        assert_equal "/bar/baz/z_category/mixedcase/2013/07/22/post-excerpt-with-layout.html", @excerpt.to_liquid["url"]
+        assert_equal "/bar/baz/z_category/mixedcase/2013/07/22/"\
+                     "post-excerpt-with-layout.html", @excerpt.to_liquid["url"]
```

Comment:
@parkr how do you feel about this?

```
        url = "/bar/baz/z_category/mixedcase/2013/07/22/post-excerpt-with-layout.html"
        assert_equal url, @excerpt.to_liquid["url"]
```
Using `expected` or `expected_url` triggers the line limit. by 1 and 5 chars respectively. Upside of this setup, one line for the path to the file. Downside, inconsistent with the naming of variables in tests.